spec/index.bs

```diff
@@ -1476,6 +1478,72 @@ On setting {{SVGAnimatedString/baseVal}}, the following steps are run:
 
 Note: SVG does not have a complete script processing model <a href="https://github.com/w3c/svgwg/issues/196">yet</a>. Trusted Types assumes that the attribute and text body modification protections behave similarly to ones for HTML scripts outlined in [[#enforcement-in-scripts]].
 
+## Integration with DOM ## {#integration-with-dom}
+
+This document modifies the following interfaces defined by [[DOM]]:
+
+<pre class="idl exclude">
+partial interface mixin ParentNode {
+  [CEReactions, Unscopable] undefined prepend((Node or DOMString or TrustedScript)... nodes);
```

Comment:
Confused: when to use `[StringContext=TrustedScript] DOMString` and when to use `DOMString or TrustedScript`?

The former is specified in https://w3c.github.io/trusted-types/dist/spec/#StringContext.

Reply:
TrustedScript is the object itself, where as `[StringContext=TrustedScript] DOMString` (or its alias `ScriptString`) pass the string itself through.

These methods need the raw object because the processing happens at the algorithm level (as it needs to process TextNode content AND strings through a policy)

Reply:
> TrustedScript is the object itself, where as `[StringContext=TrustedScript] DOMString` (or its alias `ScriptString`) pass the string itself through.
> 
> These methods need the raw object because the processing happens at the algorithm level (as it needs to process TextNode content AND strings through a policy)

Doesn't it imply `someParentNode.prepend(someString)` may be called, even if trusted types are enforced?

Reply:
The linked DOM spec PR updates the relevant algorithms so that the string gets put through the trusted types algos.